### PR TITLE
update addon status from more than one collectors

### PR DESCRIPTION
**Pull request by marcolan018**
Signed-off-by: marcolan018 <llan@redhat.com>

https://issues.redhat.com/browse/ACM-1930

### CODE REVIEW

File: collectors/metrics/pkg/status/status.go [hunk 1]
```diff
@@ -117,15 +129,42 @@ func (s *StatusReport) UpdateStatus(t string, r string, m string) error {
 			LastTransitionTime: metav1.NewTime(time.Now()),
 		})
 		update = true
+		lastestC = oav1beta1.StatusCondition{
```
Comment (dislbenn) [inline collectors/metrics/pkg/status/status.go:132]: Is `lastestC` supposed to be `latestC`?

File: collectors/metrics/pkg/status/status.go [hunk 2]
```diff
@@ -63,6 +66,10 @@ func (s *StatusReport) UpdateStatus(t string, r string, m string) error {
 	if s.statusClient == nil {
 		return nil
 	}
+	isUwl := false
+	if os.Getenv("FROM") == uwlPromURL {
+		isUwl = true
+	}
```
[the remark this answers is not in the document]
Comment (marcolan018) [reply collectors/metrics/pkg/status/status.go:72]: yes. 
It's set in endpoint operator when creating metrics collector deployment

File: collectors/metrics/pkg/status/status.go (hunk 1 above)
Comment (marcolan018) [reply collectors/metrics/pkg/status/status.go:132]: thanks, will fix the typo

**Pull request closed without merging**